src/transformer/utils/get_activity/book_chapter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace src\transformer\utils\get_activity;
+defined('MOODLE_INTERNAL') || die();
+
+use src\transformer\utils as utils;
+
+function book_chapter(array $config, \stdClass $course, \stdClass $chapter) {
+    $courselang = utils\get_course_lang($course);
+    $url = $config['app_url'].'/mod/book/view.php?id=5&chapterid='.$chapter->id;
```

Comment:
Did you intend to hard code the `5` in here for id param?

Reply:
Oops! Good spot!

Reply:
No worries, I'm actually amazed I spot that, good to know reviews actually catch some things 😆

Reply:
The tests are catching it too, but you saved me some head scratching to find it!

Reply:
Ah cool no problem, hadn't actually checked the failing build yet.